### DISCUSSION TITLE
I made a site for privacy-focussed SaaS tools and featured AppFlowy ✨

### DISCUSSION THREAD

**Discussion by fredrivett** [AppFlowy-IO/AppFlowy#992] [Show and tell]
Hey folks,

Hope it's ok to post here, noticed the `🙌 Show and tell` category and thought this would be a decent spot.

A couple months ago I was looking for **the best privacy-focussed products to build and grow my startup**, and I couldn't find a great place that was aimed at startups, so I built one myself:

📝 [PrivacyShortlist.com](https://privacyshortlist.com/)

You can filter by features, browse by category, see who has the most github stars and recommend the products you love.

❤️ Feel free to give some love to AppFlowy here: [AppFlowy | Privacy-Focussed 🧠 Knowledge base](https://privacyshortlist.com/knowledge-base/product/appflowy)

Would love any thoughts and feedback too.

Cheers!
Fred.

**Comment by lawweiliang** [comment]
@fredrivett 

Wow, there is a number of open source product that I never heard of, I will bookmark your site. 

Thanks for your privacy aggregator tool. Love it. 
To reach more users, you might try to post your product over "product hunt". https://www.producthunt.com/ 
(One of my favorites tool to hunt for new products and learn from others.  😄 😄 )

**Reply by fredrivett** [reply]
Hey Law, thanks so much for taking the time to take a look at the site and provide feedback!

Totally agree with you on the PH launch, I've launched quite a few things there before and I'm planning to do a launch in the next couple weeks, I'll announce more on the newsletter when it comes to that, hopefully AppFlowy will get some extra eyeballs too.

**Comment by annieappflowy** [comment]
@fredrivett wow, congrats on the launch. Love the theme you chose. Privacy-focused is a niche market that keeps growing! Thanks a lot for featuring AppFlowy.

Just put in my two cents as you welcome feedback. If you can allow us to sign in through GitHub accounts that would be convenient for users who come from GitHub to your site. Also, second to @lawweiliang , Product Hunt seems a good fit for marketing your site.

**Reply by fredrivett** [reply]
Hey Annie, really appreciate you taking a look at this! I'm hopeful PrivacyShortlist will be just another place where people can find great tools like this to power their startup without having to expose their users to big tech tracking.

Love the feedback on Github sign in, I agree Twitter alone isn't enough, I did it mostly for quick convenience to get something going. Another option I'm pondering is "one click email verification", where you just enter your email, click the link in it, and then you're in. Would that work for you or do you (in your honest opinion) think you'd prefer Github auth?

Cheers!

**Reply by annieappflowy** [reply]
One click email verification works for me as well as Github auth. BTW, I'm not sure why you ask ppl to register in order to vote :D If it's a cold start, I would offer the ability to vote up without registering :D

**Reply by fredrivett** [reply]
Yeah that's a fair point! I wasn't sure how to verify upvotes without registration, as otherwise people could upvote multiple times unless I do some kind of fingerprint of their device and store that, which feels a bit icky and would still mean they can upvote multiple times on different devices.

Maybe it's not that important to verify uniqueness of upvotes, what do you think?

**Reply by annieappflowy** [reply]
IMHO, I don't think people would game your system when it kicks off. You can add registration after it gets decent traffic anyways. If registration adds other value than moderation, I will make it as easy as possible. Not sure if most of your target users have Twitter accounts or are willing to register through their Twitter.

**Reply by fredrivett** [reply]
It's a totally fair point. I'll ponder this, I think I'll either add one click email verification or consider removing the requirement to authenticate at all, as I agree it seems no-one actually wants to use the Twitter auth currently.

**Reply by lawweiliang** [reply]
> One click email verification works for me as well as Github auth. BTW, I'm not sure why you ask ppl to register in order to vote :D If it's a cold start, I would offer the ability to vote up without registering :D

At first, I want to vote. As I see the register part, then I am out. 😅

**Reply by fredrivett** [reply]
Does that include any form of verification, including one click email verification @lawweiliang? Just finding it tricky to balance verified recommendations and actually making it usable.

**Reply by lawweiliang** [reply]
> Does that include any form of verification, including one-click email verification @lawweiliang? Just finding it tricky to balance verified recommendations and actually making it usable.

@fredrivett I agree with @annieappflowy, it is really weird to register an account just for voting.
At least I am not going to do that so I don't expect others will do it. 😺  

Instead of thinking about whether you need the user to register or not, the question that I will ask myself is "Would I provide any features that require the user to log in?" If I can figure it out, then include the registration part, if couldn't, remove that. Straight to the point. 

---

For the user login/authentication part, things are getting complicated nowadays. Users can use any tool that they want to log in, be it google, yahoo, Hotmail, Twitter, Facebook, etc.
Do you need to include 2fa? 
Do you have multifactor login each time user uses a different IP address?
Do you consider google passkey? (https://www.youtube.com/watch?v=U89EByn3jEY)

That is why, for this part, I will directly use a third-party company such as https://web3auth.io/ .

**Reply by fredrivett** [reply]
Yeah maybe I need to look at a way to allow users to upvote and store a fingerprint from them, so they can't just keep clicking it 100 times and it adds 100 upvotes. I'll ponder this thanks @lawweiliang.

**Reply by fredrivett** [reply]
@annieappflowy @lawweiliang hey Annie/Law, just a heads up that I've now implemented `sign in with magic one-click email link` so you can easily login without needing to set another password, this way it keeps the friction low and also makes sure each vote has some level of quality check by requiring a unique email address per vote.

let me know your honest thoughts!

**Comment by lawweiliang** [comment]
@fredrivett 
Since you are here, there is another thing that I would like to share with you.

Regarding email subscription. Once I clicked the confirmation link over the email, it redirect me back to the system using this URL "https://privacyshortlist.com/?subscribe=success" without prompting any successful message.

**Reply by fredrivett** [reply]
ah good spot @lawweiliang, thanks for the heads up, I've now fixed that and set the redirect URL to: https://privacyshortlist.com/?subscribed=true, which shows the flash message.
